dotnetv3/Glue/README.md

```diff
@@ -0,0 +1,73 @@
+# AWS Glue code examples for the AWS SDK for .NET v3
+
+## Overview
+
+Examples in this folder show how to use AWS Glue with the AWS SDK for .NET.
```

Comment:
The examples in this folder show you how to use AWS Glue with the AWS SDK for .NET.

Reply:
changed